src/dispatch/plugins/dispatch_slack/service.py

```diff
@@ -246,7 +246,11 @@ def add_users_to_conversation_thread(
     client: WebClient, conversation_id: str, thread_id, user_ids: List[str]
 ) -> NoReturn:
     """Adds user to a threaded conversation."""
-    users = [f"<@{user_id}>" for user_id in user_ids]
```

Comment:
If we don't care about the ordering can we call `set()` here to dedupe instead?

```python
users = [f"<@{user_id}>" for user_id in set(user_ids)]
```

Reply:
Yes, I can use `set`. I just wanted to make it more explicit. I've actually id'ed other places where we want to dedupe as well.